src/main/java/gregtech/api/fluids/fluidType/FluidTypes.java

```diff
@@ -0,0 +1,9 @@
+package gregtech.api.fluids.fluidType;
+
+public class FluidTypes {
+
+    public static final FluidType LIQUID = new FluidTypeLiquid("liquid", "", "", "gregtech.fluid.generic");
+    public static final FluidType ACID = new FluidTypeAcid("acid", "", "", "gregtech.fluid.generic");
+    public static final FluidType GAS = new FluidTypeGas("gas", "", "", "gregtech.fluid.generic");
+    public static final FluidType PLASMA = new FluidTypePlasma("plasma", "plasma", "", "gregtech.fluid.plasma");
+}
```

Comment:
Can make this a CT class as well, so you can get these stock FluidTypes easily from CT.